src/applications/vre/28-1900/config/chapters/veteran/veteran-information/veteranInformation.js

```diff
@@ -41,7 +42,14 @@ export const uiSchema = {
       ...ssnUI,
     },
     vaFileNumber: {
-      'ui:title': 'Your VA file number (*If different from SSN)',
+      'ui:title': (
+        <p className="vads-u-margin--0 vads-u-margin-top--3 vads-u-display--inline-block vads-u-font-weight--normal vads-u-color--base vads-u-font-family--sans vads-u-font-size--base">
+          Your VA file number{' '}
+          <span className="schemaform-required-span">
+            (*If different from SSN)
```

Comment:
Does the `SSN` abbreviation need to be spelled out? Something like:

```html
<dfn>
  <abbr title="social security number">SSN</abbr>
</dfn>
```

Reply:
This is correct. Code would be:

```
<span className="schemaform-required-span">
            (*If different from <abbr title="social security number">SSN</abbr>)
```